go.mod

```diff
@@ -1,78 +1,113 @@
 module istio.io/bots
 
-go 1.18
+go 1.21
+
+toolchain go1.21.4
```

Comment:
Why do we need this? Does this enforce everyone is using 1.21.4?

Reply:
Interestingly, that came in as part of the `go mod tidy`. Will remove that.

Still chasing why the license files generated from my `make gen` locally are different than what the pipeline generates. This is the second item I've found is different, the first being sort, when using the build-tools image on my local machine vs the pipeline.